ptsrc/parser/stdIdentifiers

```diff
@@ -39,7 +41,7 @@ repeat
 
 % Predeclared type names
 integer
-char
+string
```

Comment:
This is an interesting one. It doesn't seem to be used anywhere else, so I haven't changed anything more yet. Will do some testing